jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -11,34 +11,164 @@ contentTags:
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_go-to-definition-optimised.gif[Screenshot showing the definition available on hover]
+
+- **Contextual documentation and usage hints when hovering on specific keys.** This helps you avoid having to frequently switch to your browser to check the documentation when editing your configuration. Links to the official CircleCI docs are also provided on hover, for easier navigation.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_on-hover-documentation.png[Screenshot showing the contextual information on hover]
+
+- **Syntax validation**. This helps you identify typos, incorrect use of parameters, incomplete definitions, wrong types, invalid or deprecated machine versions, etc.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_syntax-validation.gif[Screenshot showing the synthax highlightning when an error is identified]
+
+- **Usage warnings**. This helps you identify deprecated parameters, unused jobs or executors, or missing properties that prevent you from taking advantage of CircleCI’s full capabilities.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_usage-warning.png[Screenshot showing code highlightning to warn on an unused job]
+
+- **Auto completion**. This is available with both built-in keys and parameters as well as user-defined variables.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_config_helper_autocomplete.png[Screenshot showing two suggestions to autocomplete the line of code]
+
+[#config-validation-commands]
+=== Config validation commands
+
+The config helper also provides two commands that help you statically validate your YAML config files without having to run a pipeline.
+
+* Validate current configuration file
++
+Corresponds to the CLI command `circleci config validate`, and verifies statically that the config file is well formed. Please note that this command only validates this file for structure and syntax errors, but not for semantic error (e.g. "This job does not exist").
+
+* Validate current configuration file against org policy
++
+Corresponds to the CLI command `circleci policy decide`, and verifies that the configuration file complies with your organisation policies (if any are set).
+
+Both of these commands can be invoked by:
+
+- Right-clicking on a CircleCI YAML file.
+
+- Clicking on the CircleCI button on the top-right corner of a tab in which a `.circleci/config.yml` is open. Note that the button will not be visible if you are editing any other file.
+
+[#open-source-language-server]
+=== Open source language server
+
+The config helper is based on a dedicated language server specific for CircleCI YAML files, and is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[circleci-yaml-language-server].
```

Comment:
_which_ is open source (otherwise it refers to the Config Helper, which is the name we give to its capabilities inside VS Code... sorry, it's a bit confusing)